src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -238,7 +244,7 @@ protected int getWarehouseCount(ItemStorage item)
 
```

Comment:
while you're at it, can you change the above warehouse search to use the building managers saved warehouse list?

Reply:
done